extensions.yaml

```diff
@@ -4,6 +4,7 @@
 
 repos:
   - rhel-8-advanced-virt
+  - rhel-8-appstream
```

Comment:
You can drop this repo from this manifest; `rpm-ostree` will use the repos in the `manifest.yaml` config for finding RPMs.  This list is better thought of as "I need these additional repos for extensions"

Reply:
Perfect.  Do I also have to add `http://rhsm-pulp.corp.redhat.com/content/dist/layered/rhel8/x86_64/rhocp/4.8/` in the same way that it was done for the advanced-virt one?

Asking because the `kata-containers` RPM will show up there (not yet released, thus not yet there).

Reply:
I think `rhocp` is the same as the RHOSE one?

https://github.com/openshift/release/blob/db8e82f518f55d46d80493744d074e5d657b2259/core-services/release-controller/_repos/ocp-4.8-rhel8.repo#L25-L28

Reply:
Maybe not; probably worth asking the ART or release folks

Reply:
Yep, not sure yet. :-)
But AFAIR, it'll be released in a "tech-preview" kind of repo / CDN, so, if needed, I'll add the needed bits to `redhat-cores` as already done for the advanced-virt repo, and propagate it here accodingly.

@miabbott, thanks a lot for the pointers and the review, very much appreciated!